src/features/projectsV2/ProjectDetails/styles/Slider.module.scss

```
@@ -41,6 +41,28 @@
   }
 }
 
+// Fill the MUI modal so it takes up the full viewport. This keeps the image
+// container and slider buttons inside a full-screen flex layout. Without it,
+// the container would size itself to its content, preventing it from shrinking
+// below 800px and causing the mobile arrow buttons to appear outside the screen.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)